python/src/com/jetbrains/python/psi/impl/PyImportedModule.java

```diff
@@ -111,6 +116,21 @@ public PsiElement resolve() {
     return element;
   }
 
+  @NotNull
+  public List<PsiElement> multiResolve() {
+    final List<RatedResolveResult> results;
+    if (myImportElement != null) {
+      results = ResolveImportUtil.multiResolveImportElement(myImportElement, myImportedPrefix);
+    }
+    else {
+      results = ResolveResultList.to(ResolveImportUtil.resolveModuleInRoots(myImportedPrefix, myContainingFile));
+    }
+
+    return results.stream().map(res -> res.getElement())
+                  .map((PsiElement el) -> el instanceof PsiDirectory ? PyUtil.getPackageElement((PsiDirectory)el, this) : el)
+                  .collect(Collectors.toList());
```

Comment:
Consider using `StreamEx` library, it has useful methods for streams.
E.g. `collect(Collectors.toList())` could be rewritten as `toList()`.